source/includes/fact-create-text-index.rst

```diff
@@ -1,9 +1,5 @@
-To perform text search queries, you must have a
-``text`` index on your collection. A collection can only have **one**
-text search index, but that index can cover multiple fields.
-
-For example you can run the following in :binary:`~bin.mongosh` to
-allow text search over the ``name`` and ``description`` fields:
+You can run the following in :binary:`~bin.mongosh` to allow text 
```

Comment:
Suggest removing "You can" and being more direct. "Run the following command in...